src/ILCompiler.Compiler/src/Compiler/SimdHelper.cs

```diff
@@ -34,8 +34,8 @@ public bool IsSimdType(TypeDesc type)
                 if (metadataType.IsIntrinsic)
                 {
                     string name = metadataType.Name;
-                    if ((name == "Vector`1" || name == "Vector") &&
-                        metadataType.Namespace == "System.Numerics")
+                    if (((name == "Vector") || (name == "Vector`1") || (name == "Vector2") || (name == "Vector3") || (name == "Vector4")) &&
```

Comment:
CoreFX update may be useful as well. RyuJIT is not always happy when it sees two copies of the Vector intrinsic types.

Reply:
Good point - updated that too.